README.md

```diff
@@ -52,6 +53,7 @@ custom:
 | Option                    | Description                                                                                                           | Default                                                                    | Example                                                  |
 | ------------------------- | --------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------- | -------------------------------------------------------- |
 | `generateSwaggerOnDeploy` | Boolean which indicates whether to generate a new swagger file on deployment                                          | `true`                                                                     |                                                          |
+| `title`                   | String to overwrite the project title with a custom one                                                               | `-`                                                                        |                                                          |
```

Comment:
Instead of `` `-` `` for the title default, we should probably put something like `Serverless service name` so it's clear where this library pulls the default name from